### PR TITLE
Ignoring touch events generated by HW mouse

### PR DESCRIPTION
**Pull request by gilnoh**
This is a patch for issue #114 . 

- Since commit 6199841, HW mouse events are handled by GCMouse. 
- However, HW mouse clicks still generate (additional) touch events. These touch events are then coming to and handled by touch handling code. This causes issues reported in #114, such as additional left clicks or jumping mouse pointers.  
- In this patch, I added a small code that checks touch events for their origin. So it can ignore events from HW pointing device. 
- Maybe there is a better way to handle this: For example, somehow disable mouse clicks from generating any touch event at all, would be a better way to handle this. However, I can't seem to find a way to do this.

### CODE REVIEW

File: SDL/src/video/uikit/SDL_uikitview.m [hunk 1]
```diff
@@ -203,6 +210,13 @@ - (void)touchesEnded:(NSSet *)touches withEvent:(UIEvent *)event
 {
 	for (UITouch *touch in touches)
 	{
+        if (@available(iOS 13.4, *)) {
+            if (touch.type == UITouchTypeIndirectPointer) {
+                // ignore touch event generated by HW mouse
+                return;
```
Comment (gilnoh) [inline SDL/src/video/uikit/SDL_uikitview.m:216]: Maybe this should be continue; ...

Comment (gilnoh) [inline SDL/src/video/uikit/SDL_uikitview.m:216]: maybe should be continue, instead of return ... hmm.

**Comment by litchie** [comment]
Thanks for fixing this issue. Looks good

**Comment by gilnoh** [comment]
My pleasure!

**Pull request merged**